docs/integrations/sources/facebook-marketing-migrations.md

```diff
@@ -1,5 +1,17 @@
 # Facebook Marketing Migration Guide
 
+## Upgrading to 4.0.0
+
+Streams Ads-Insights-* no longer use the action_report_time parameter.
+
+### Update Custom Insights Reports (this step can be skipped if you did not define any)
+
+1. Select **Sources** in the main navbar.
+   1. Select the Facebook Marketing Connector.
+2. Select the **Retest saved source**.
+3. Remove unsupported fields from the list in Custom Insights section.
+4. Select **Test and Save**.
+
```

Comment:
If we can implement a config migration or make non-breaking by simply hiding the config in the UI, then these steps could be removed.

```suggestion
```